app/assets/stylesheets/bulk-actions.scss

```diff
@@ -0,0 +1,10 @@
+.new_bulk_action {
+  .tab-pane {
+    @extend .mt-3;
+    @extend .mb-3;
+  }
+
+  .help-block {
+    font-weight: bold;
+  }
+}
```

Comment:
And I assume everywhere these are used they're inside the `.new_bulk_action` context? 👍🏻

Reply:
I believe that is the case